next.config.js

```diff
@@ -13,15 +11,15 @@ const nextConfig = {
   },
 }
 
-const sentryWebpackPluginOptions = {
-  // For all available options, see:
-  // https://github.com/getsentry/sentry-webpack-plugin#options.
+const sentryConfig = {
+  org: 'awell',
+  project: 'hosted-pages',
+  authToken: process.env.SENTRY_AUTH_TOKEN,
+  silent: false,
+  hideSourceMaps: true,
 }
 
-const nextPlugins = [
-  withImages,
-  (nextConfig) => withSentryConfig(nextConfig, sentryWebpackPluginOptions),
-]
+const nextPlugins = [(nextConfig) => withSentryConfig(nextConfig, sentryConfig)]
```

Comment:
why did you remove `withImages` plugin ?

Reply:
I haven't seen any usage of it in the project. Plus nextjs has na build-in `<Image />` component that we should use for optimization.